components/demo/demo-snippet.js

```diff
@@ -152,15 +148,10 @@ class DemoSnippet extends LitElement {
 		changedProperties.forEach((_, prop) => {
 			if (prop === '_code') {
 				if (this.shadowRoot) this.shadowRoot.querySelector('d2l-code-view').forceUpdate();
-				this._updateHasSkeleton();
```

Comment:
This was causing one of the duplicate render calls in `d2l-demo-snippet`. This is simply moved since we don't really need to wait for Lit to update to update the skeleton bit.